input/pagecontent/security_recommendations.md

```diff
@@ -0,0 +1,49 @@
+This page describes the recommendations for a minimum level of security when implementing the BFDR IG. The [FHIR Security](https://www.hl7.org/fhir/security.html) web page defines more data exchange protocols and content models. BFDR systems should refer to that if a higher level of security is required for interoperability. At minimum, two areas need protection in the data exchange:
```

Comment:
Is this lifted from somewhere?   Seems like it could just reference some other sources and call it a day.
What about this is specific to our use cases?